force-app/main/default/lwc/link/link.js

```diff
@@ -35,20 +47,33 @@ export default class Link extends LightningElement {
     setCurrentPath() {
         this.currentPath = this.routerInstance.currentPath;
     }
-    toggleActiveAttribute() {
+    toggleActiveAttributes() {
         if (this.active) {
             this.setAttribute("active", "");
+            this.applyActiveClassTo(this._slottedElement);
         } else {
             this.removeAttribute("active");
+            this.removeActiveClassFrom(this._slottedElement);
         }
     }
+    applyActiveClassTo(element) {
+        if (!element) return;
+        this.activeClass.forEach(activeClass => element.classList.add(activeClass));
+    }
+    removeActiveClassFrom(element) {
+        if (!element) return;
+        this.activeClass.forEach(activeClass => element.classList.remove(activeClass));
+    }
     handleClick(e) {
         e.stopPropagation();
         if (this.to) {
             this.routerInstance.currentPath = this.to;
             this.currentPath = this.to;
         }
     }
+    renderedCallback() {
+        this._slottedElement = this.querySelector('*');
```

Comment:
In my case, `this._slottedElement` always blank.

![image](https://user-images.githubusercontent.com/27341550/92748478-d27ed380-f3a2-11ea-9003-e33e41a1b1b5.png)

Reply:
This brings up a good question for you.  
This PR only sets the active class on content that is slotted into the link component, but not on the default lightning-button.  Given that the lightning-button component is very opinionated and is styled with the variant prop it seemed like it was less useful to set a class on it.  So _slottedElement will only have a value if theres something inserted in the slot.
What behavior do you think is correct here?  Should we have the active classes applied to the default lightning-button component?

Reply:
You are right, It's working perfectly with the slotted into the link component. 
I thing, It's fine for default one.

Changes looks awesome. Thanks